bugsnag-plugin-android-ndk/src/main/java/com/bugsnag/android/NdkPlugin.kt

```diff
@@ -66,12 +69,60 @@ internal class NdkPlugin : Plugin {
         }
     }
 
+    fun setInternalMetricsEnabled(enabled: Boolean) {
+        nativeBridge?.setInternalMetricsEnabled(enabled)
+    }
+
     fun getSignalUnwindStackFunction(): Long {
-        val bridge = nativeBridge
-        if (bridge != null) {
-            return bridge.getSignalUnwindStackFunction()
+        return nativeBridge?.getSignalUnwindStackFunction() ?: 0
+    }
+
+    fun getCurrentCallbackSetCounts(): Map<String, Int> {
+        return nativeBridge?.getCurrentCallbackSetCounts() ?: mapOf()
+    }
+
+    fun getCurrentNativeApiCallUsage(): Map<String, Boolean> {
+        return nativeBridge?.getCurrentNativeApiCallUsage() ?: mapOf()
+    }
+
+    fun initCallbackCounts(counts: Map<String, Int>) {
+        nativeBridge?.initCallbackCounts(counts)
+    }
+
+    fun notifyAddCallback(callback: String) {
+        nativeBridge?.notifyAddCallback(callback)
+    }
+
+    fun notifyRemoveCallback(callback: String) {
+        nativeBridge?.notifyRemoveCallback(callback)
+    }
+
+    fun setStaticData(data: Map<String, Any>) {
+        var writer: Writer? = null
```

Comment:
```
val staticDataString = StringWriter().apply { use { writer -> JsonStream(writer).use { it.value(data) } } }.toString()
```

it might need a `runCatching` or similar to swallow any possible exceptions, but `use` is favourable to `try {} catch {} finally {}`